packages/iris-grid/src/sidebar/ChartBuilder.test.jsx

```diff
@@ -12,152 +13,114 @@ function makeChartBuilderWrapper({
     IrisGridTestUtils.makeTable(COLUMN_NAMES.map(IrisGridTestUtils.makeColumn))
   ),
 } = {}) {
-  return mount(
+  return render(
     <ChartBuilder onChange={onChange} onSubmit={onSubmit} model={model} />
   );
 }
 
 it('renders without crashing', () => {
-  const wrapper = makeChartBuilderWrapper();
-  wrapper.unmount();
+  const { unmount } = makeChartBuilderWrapper();
+  unmount();
 });
 
 it('updates the chart type', () => {
-  const wrapper = makeChartBuilderWrapper();
+  const { container, unmount } = makeChartBuilderWrapper();
 
-  let chartButtons = wrapper.find('.btn-chart-type');
-  expect(wrapper.state('type')).toBe(ChartBuilder.types[0]);
-  expect(chartButtons.at(0).hasClass('active')).toBe(true);
-  expect(chartButtons.at(1).hasClass('active')).toBe(false);
+  let chartButtons = container.querySelectorAll('.btn-chart-type');
 
-  chartButtons.at(1).simulate('click');
+  expect(chartButtons[0].classList.contains('active')).toBe(true);
+  expect(chartButtons[1].classList.contains('active')).toBe(false);
 
-  chartButtons = wrapper.find('.btn-chart-type');
+  userEvent.click(chartButtons[1]);
 
-  expect(wrapper.state('type')).toBe(ChartBuilder.types[1]);
-  expect(chartButtons.at(0).hasClass('active')).toBe(false);
-  expect(chartButtons.at(1).hasClass('active')).toBe(true);
+  chartButtons = container.querySelectorAll('.btn-chart-type');
 
-  wrapper.unmount();
+  expect(chartButtons[0].classList.contains('active')).toBe(false);
+  expect(chartButtons[1].classList.contains('active')).toBe(true);
+
+  unmount();
 });
 
 it('has x-axis selection with the proper columns', () => {
-  const wrapper = makeChartBuilderWrapper();
+  const { container, unmount } = makeChartBuilderWrapper();
+  const xAxisSelectElm = container.querySelector('.select-x-axis');
 
-  expect(wrapper.find('.select-x-axis').children().length).toBe(
+  expect(xAxisSelectElm.querySelectorAll('option').length).toBe(
     COLUMN_NAMES.length
   );
-  expect(wrapper.find('.select-x-axis').instance().value).toBe(COLUMN_NAMES[0]);
-  expect(wrapper.state('xAxis')).toBe(COLUMN_NAMES[0]);
-
-  wrapper.find('.select-x-axis').childAt(1).simulate('change');
```

Comment:
We should still have the change happen as a user event, see if the column names get updated